lib/cucumber/runtime.rb

```diff
@@ -30,8 +30,12 @@ def initialize(original_exception, path)
   class FileNotFoundException < FileException
   end
 
-  class FeatureFolderNotFoundException < FileException
-    include FixRuby21Bug9285 if Cucumber::RUBY_2_1 || Cucumber::RUBY_2_2 || Cucumber::RUBY_2_3
+  class FeatureFolderNotFoundException < Exception
+    attr :path
+    def initialize(path)
+      super('No such file or directory - features')
+      @path = path
+    end
   end
```

Comment:
@nodo because of fixing this issue https://github.com/cucumber/cucumber-ruby/issues/1071#issuecomment-276709766 (Reason of the issue is https://bugs.ruby-lang.org/issues/9285)

There are 2 possible ways to to fix this issue. And @mattwynne prefer "1".
1. Adding `Cucumber::RUBY_2_3` in above logic.
2. Not extend `FileException`